orchagent/aclorch.cpp

```diff
@@ -1177,6 +1177,47 @@ bool AclTable::create()
     return status == SAI_STATUS_SUCCESS;
 }
 
+void AclTable::update(SubjectType type, void *cntx)
+{
+    SWSS_LOG_ENTER();
+
+    // Only interested in port change
+    if (type != SUBJECT_TYPE_PORT_CHANGE)
+    {
+        return;
+    }
+
+    PortUpdate *update = static_cast<PortUpdate *>(cntx);
+
+    Port port = update->port;
+    if (update->add)
+    {
+        if (pendingPortSet.find(port.m_alias) != pendingPortSet.end())
+        {
+            sai_object_id_t bind_port_id;
+            if (gPortsOrch->getAclBindPortId(port.m_alias, bind_port_id))
+            {
+                link(bind_port_id);
+                bind(bind_port_id);
+
+                SWSS_LOG_NOTICE("Bound port %s to ACL table %s",
```

Comment:
Should we remove the port from pendingPortSet after successfully bind?

Reply:
line 1203

---
In reply to: [234106802](https://github.com/Azure/sonic-swss/pull/689#discussion_r234106802) [](ancestors = 234106802)

Reply:
It was not there in the original code, but now looks good.